src/components/AppBreadcrumbs/AppBreadcrumbs.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import { Breadcrumbs, Link } from '@material-ui/core';
+import { Link as RouterLink } from 'react-router-dom';
+
+function capitalize(str: string): string {
+  return str.charAt(0).toUpperCase() + str.slice(1);
+}
+
+const styles = (theme: any) => ({
+  breadcrumb: {
+    fontStyle: 'italic',
```

Comment:
why italics?  I don't think the designs have the italics (also, I like it better without them :) )

Reply:
Whoops. 1998 Trevyn made an appearance and italicized the links!